paddle/framework/net.h

```diff
@@ -103,18 +79,10 @@ class Net {
 class PlainNet : public Net {
```

Comment:
since we have reached an agreement in treating net as a composed Op, seems we do not need a `PlainNet` anymore. We can have two types of Net, `Net`, `DAGNet`.